packages/workflow/src/NodeHelpers.ts

```diff
@@ -1849,9 +1849,6 @@ export function getParameterIssues(
 				propertyOptions.name,
 				basePath.slice(0, -1),
 			);
-			if (value === undefined) {
-				continue;
```

Comment:
Is there any case where this could cause problems? We're now validating child properties of empty `fixedCollections` if I follow the code

Reply:
I didn't see any issues so far, this was added since `value` will be undefined when no fields are set and we need to check for minimum filed limit at this point. I can move the check upward in the function and skip rest of it like we do right now. WDYT @elsmr ?

Reply:
Though a bit more about it and it makes sense to add bac